### PR TITLE
Feature/104(会員情報表示)

### PR DESCRIPTION
**Pull request by uedamiku**

### CODE REVIEW

File: src/main/java/com/example/controller/AdminController.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 import com.example.domain.Item;
 import com.example.domain.LoginUser;
+import com.example.domain.User;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.User' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/AdminController.java [hunk 2]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
 		adminService.addItem(item, form.getImage());
 		return "redirect:/admin/show-item-list";
 	}
+
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:80]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/AdminController.java [hunk 3]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
 		adminService.addItem(item, form.getImage());
 		return "redirect:/admin/show-item-list";
 	}
+	
+	@GetMapping("/show-user-list")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:81]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:81]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:81]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/controller/AdminController.java [hunk 4]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+	public String showUserList(Model model) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:82]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/AdminController.java [hunk 5]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+		List<User> userList = adminService.showUserList();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:83]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:83]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 6]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+		model.addAttribute("userList", userList);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:84]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:84]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 7]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+		System.out.println(userList);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:85]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:85]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 8]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+		return "admin/user_list";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:86]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:86]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 9]
```diff
@@ -76,4 +77,12 @@ public String AddItem(InsertItemForm form) {
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:87]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:87]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/repository/UserRepository.java [hunk 10]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
 		return userList.get(0);
 	}
 
+	/**
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/UserRepository.java [hunk 11]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+	 * 全ユーザ情報を取得します.
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/UserRepository.java [hunk 12]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+	 *
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/UserRepository.java [hunk 13]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
 		return userList.get(0);
 	}
 
+	/**
+	 * 全ユーザ情報を取得します.
+	 * 
+	 * @return ユーザ情報
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@return' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/UserRepository.java [hunk 14]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+	 */
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/UserRepository.java [hunk 15]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+	public List<User> findAll() {
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/repository/UserRepository.java [hunk 16]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+
+		String sql = "SELECT id, name, email, zipcode, pref, municipalities, address, telephone FROM users WHERE role_admin=0 ORDER BY id;";
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 148).

Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/UserRepository.java [hunk 17]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+
+		List<User> userList = template.query(sql.toString(), USER_ROW_MAPPER);
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:73]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:73]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/UserRepository.java [hunk 18]
```diff
@@ -61,4 +61,17 @@ public User findByEmail(String email) {
...
+		return userList;
```
Comment (tkeshun) [inline src/main/java/com/example/repository/UserRepository.java:74]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by tkeshun — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/com/example/repository/UserRepository.java|74 col 17|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/repository/UserRepository.java#L74) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/UserRepository.java|75 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/repository/UserRepository.java#L75) Line contains a tab character.
[src/main/java/com/example/repository/UserRepository.java|75 col 9|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/repository/UserRepository.java#L75) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/service/AdminService.java|14 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L14) Wrong lexicographical order for 'com.example.domain.User' import. Should be before 'org.springframework.web.multipart.MultipartFile'.
[src/main/java/com/example/service/AdminService.java|17 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L17) Wrong lexicographical order for 'com.example.repository.UserRepository' import. Should be before 'org.springframework.web.multipart.MultipartFile'.
[src/main/java/com/example/service/AdminService.java|32 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L32) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|32 col 9|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L32) 'member def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/service/AdminService.java|33 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L33) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|45 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L45) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|46 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L46) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|47 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L47) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|48|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L48) Javadoc tag '@return' should be preceded with an empty line.
[src/main/java/com/example/service/AdminService.java|48 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L48) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|49 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L49) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|50 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L50) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|50 col 9|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L50) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/service/AdminService.java|51 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L51) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|51 col 17|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L51) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/service/AdminService.java|52 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L52) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|52 col 17|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L52) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/service/AdminService.java|53 col 1|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L53) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|53 col 9|](http://github.com/ecsite-project/ec-202304c/blob/47f06cee8221a58b3f5dc8789fd70dfdaeeafc58/src/main/java/com/example/service/AdminService.java#L53) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
</details>

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**